README.md

```diff
@@ -31,18 +50,12 @@ In each template we've also provided boilerplate examples for the additional sec
 - **Official Communication Channels** (Required) – A full list of the communication methods that are made available by the organization to receive and communicate about vulnerability submissions.
 - **Disclosure Policy** (Required) – A clear policy outlining the conditions under which a researcher can disclose the details of a reported issue to third parties. 
 
-### Disclosure types
-
-- **Coordinated Disclosure**: A researcher can share details of the vulnerability after a fix has been applied and the program owner has provided permission to disclose or after a clearly-stated time has passed from submission, whichever is sooner;
-- **Discretionary Disclosure**: The researcher or the program owner can request mutual permission to share details of the vulnerability after approval is explicitly received; or
-- **Non-Disclosure**: Researchers are required to keep vulnerability details and the existence of the program itself confidential.
-
 ## Next steps 
 
 Once you've published your policy, you can:  
 - Add the appropriate [disclose.io seal](https://github.com/disclose/dioseal) to your public program brief,
 - Submit a pull request to add your program to the [open-source disclose.io program list](https://github.com/disclose/diodb),
-- Let the world know you're joining the movement!
+- Let the world know you're joining the initiative!
```

Comment:
Feels more akin to a coalition.  An alliance for combined action on VDP / BBP efforts.  My personal experience is that one forms a coalition in response to opportunities and threats.

Reply:
I'm a big fan of this... Good add.